server/views/pages/visit/summary.njk

```diff
@@ -19,7 +19,7 @@
 {% endmacro %}
 
 {% set pageHeaderTitle = "Visit booking details" %}
-{% set pageTitle = applicationName + " - " + pageHeaderTitle + " - " + visit.reference %}
+{% set pageTitle = pageHeaderTitle + " - " + applicationName + " - " + visit.reference %}
```

Comment:
```suggestion
{% set pageTitle = pageHeaderTitle + " (" + visit.reference + ") - " + applicationName  %}
```
Not sure if this is right as you get a title like `Visit booking details - Manage prison visits - za-lc-ev-mv - DPS`.

Suggestion would render as: `Visit booking details (za-lc-ev-mv) - Manage prison visits - DPS` 🤷‍♂️